### PR TITLE
small fixes + fix TODOs

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: scripts/scil_merge_sh.py [hunk 1]
```diff
@@ -33,42 +32,40 @@
...
-    # TODO Rename variable in_*
-    parser.add_argument('sh_files', nargs="+",
-                        help='List of SH files.')
-    parser.add_argument('out_sh',
-                        help='output SH file.')
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__, epilog=EPILOG)
 
-    add_overwrite_arg(parser)
+    p.add_argument('in_SHs', nargs="+",
```
Comment (GuillaumeTh) [inline scripts/scil_merge_sh.py:38]: I think we should not use capital letters for variables.

File: scripts/scil_run_nlmeans.py [hunk 2]
```diff
@@ -25,40 +27,30 @@ def _build_arg_parser():
...
-        '--processes', dest='nbr_processes', metavar='int', type=int,
-        help='Number of sub processes to start. Default: Use all cores.')
-    p.add_argument(
-        '-v', '--verbose', action="store_true", dest="verbose",
-        help="Print more info. Default : Print only warnings.")
+    p.add_argument('in_image',
+                   help='Path of the image file to denoise.')
+    p.add_argument('out_image',
+                   help='Path to save the denoised image file.')
+    p.add_argument('N', metavar='number_coils', type=int,
```
Comment (GuillaumeTh) [inline scripts/scil_run_nlmeans.py:34]: I think we should not use capital letters for variables.

File: scripts/scil_compute_fodf_max_in_ventricles.py [hunk 3]
```diff
@@ -33,30 +33,29 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
                                 description=__doc__, epilog=EPILOG)
 
-    p.add_argument('input',  metavar='fODFs',
+    p.add_argument('in_fODFs',  metavar='fODFs',
```
Comment (GuillaumeTh) [inline scripts/scil_compute_fodf_max_in_ventricles.py:36]: I think we should not use capital letters for variables.

File: scripts/scil_flip_volume.py [hunk 4]
```diff
@@ -52,7 +51,7 @@ def main():
     if 'z' in args.axes:
         data = data[:, :, ::-1, ...]
 
-    nib.save(nib.Nifti1Image(data, affine, header), args.output)
+    nib.save(nib.Nifti1Image(data, affine, header), args.out_image)
```
Comment (frheault) [inline scripts/scil_flip_volume.py:54]: Since header is optional, I prefer header=header

File: scripts/scil_merge_sh.py [hunk 5]
```diff
@@ -33,42 +32,40 @@
...
+        im = nib.load(sh_file)
         im_dat = im.get_fdata(dtype=np.float32)
 
         out_coeffs = np.where(np.abs(im_dat) > np.abs(out_coeffs),
                               im_dat, out_coeffs)
 
     # TODO remove header or add optional argument name
-    nb.save(nb.Nifti1Image(out_coeffs, first_im.affine, first_im.header),
-            args.out_sh)
+    nib.save(nib.Nifti1Image(out_coeffs, first_im.affine, first_im.header),
```
Comment (frheault) [inline scripts/scil_merge_sh.py:67]: Same as above, header is optional so I like to user the named parameter

File: scripts/scil_reorder_dwi_philips.py [hunk 6]
```diff
@@ -92,19 +85,19 @@ def main():
...
 
     assert_inputs_exist(parser, required_args)
     assert_outputs_exist(parser, args, output_filenames)
 
-    oTable = np.loadtxt(args.table, skiprows=1)
-    bvals, bvecs = read_bvals_bvecs(args.bval, args.bvec)
-    dwis = nb.load(args.dwi)
+    oTable = np.loadtxt(args.in_table, skiprows=1)
+    bvals, bvecs = read_bvals_bvecs(args.in_bval, args.in_bvec)
+    dwis = nb.load(args.in_dwi)
```
Comment (frheault) [inline scripts/scil_reorder_dwi_philips.py:100]: nb. -> nib.
(like elsewhere)

File: scripts/scil_run_nlmeans.py [hunk 7]
```diff
@@ -96,7 +88,7 @@ def main():
     if args.logfile is not None:
         log.addHandler(logging.FileHandler(args.logfile, mode='w'))
 
-    vol = nb.load(args.input)
+    vol = nb.load(args.in_image)
     data = vol.get_fdata(dtype=np.float32)
```
Comment (frheault) [inline scripts/scil_run_nlmeans.py:92]: Isn't that the default?

File: scripts/scil_run_nlmeans.py [hunk 8]
```diff
@@ -120,11 +112,11 @@ def main():
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", category=DeprecationWarning)
         data_denoised = nlmeans(
-            data, sigma, mask=mask, rician=args.N > 0,
+            data, sigma, mask=mask, rician=args.number_coils > 0,
             num_threads=args.nbr_processes)
 
     nb.save(nb.Nifti1Image(
```
Comment (frheault) [inline scripts/scil_run_nlmeans.py:118]: nb. -> nib.
Like elsewhere?

File: scripts/scil_run_nlmeans.py [hunk 9]
```diff
@@ -120,11 +112,11 @@ def main():
...
-        data_denoised, vol.affine, vol.header), args.output)
+        data_denoised, vol.affine, vol.header), args.out_image)
```
Comment (frheault) [inline scripts/scil_run_nlmeans.py:119]: header=vol.header

**Review by frheault — APPROVED** [review]

File: scripts/scil_run_nlmeans.py (hunk 7 above)
Comment (arnaudbore) [reply scripts/scil_run_nlmeans.py:91]: Yes but to make it clear in all scripts we ask to be explicit about this.

**Pull request merged**